src/test/java/uk/gov/pay/adminusers/resources/RequestValidationsTest.java

```diff
@@ -41,19 +41,51 @@ public void checkIfExists_shouldFail_whenFieldsAreProvidedWithEmptyValues() thro
     }
 
     @Test
-    public void checkIfExists_shouldFail_whenFieldsAreProvidedWithNullValues() throws Exception {
+    public void checkIfExistsOrEmpty_shouldFail_whenFieldsAreProvidedWithNullValues() {
         ObjectNode payload = JsonNodeFactory.instance.objectNode();
         payload.set(FIELD_1, null);
         payload.set(FIELD_2, null);
-        System.out.println(payload);
 
         Optional<List<String>> errors = requestValidations.checkIfExistsOrEmpty(payload, FIELD_1, FIELD_2);
 
         assertThat(errors.isPresent(), is(true));
     }
 
     @Test
-    public void checkNotBoolean_shouldSuccess_whenFieldsAreTrueOrFalse() {
+    public void checkIfExists_shouldSucceed_whenFieldsAreProvidedWithValues() {
+        ObjectNode payload = JsonNodeFactory.instance.objectNode();
+        payload.put(FIELD_1, "value1");
+        payload.put(FIELD_2, "value2");
+
+        Optional<List<String>> errors = requestValidations.checkIfExists(payload, FIELD_1, FIELD_2);
+
+        assertThat(errors.isPresent(), is(false));
+    }
+
+    @Test
+    public void checkIfExists_shouldSucceed_whenFieldsAreProvidedWithEmptyValues() {
+        ObjectNode payload = JsonNodeFactory.instance.objectNode();
+        payload.put(FIELD_1, "");
+        payload.put(FIELD_2, "");
```

Comment:
how about making FIELD_2 have some whitespace, e.g. `" "`?

Reply:
Good idea: we’ll do this.